### ISSUE TITLE
When replying to Twilio messages, the Author field is unset so messages do not get sent

### ISSUE THREAD

**Issue by dge9788** [ithunt/asksg#30]
Relevant message in the logs:

```
2013-04-02 16:18:47, DEBUG, org.apache.http.wire, http-bio-8080-exec-1, wire << "{"status":400,"message":"A 'To' phone number is required.","code":21604,"more_info":"http:\/\/www.twilio.com\/docs\/errors\/21604"}"
```

Using the debugger on the UI shows that the author field isn't set when the function $scope.doPostMessage(convo, message, author) is called when the submit button is pushed.

**Comment by dge9788** [comment]
Should be fixed, need code review then merge to master.

**Comment by rpk7524** [comment]
I think I'm going to move this up from message service into the controller.  MessageService should get a correct object, not be responsible for pulling one.  Also attached will be fixing the author setting from the ui.  getRecipient will be a getter implementation, I think we always fetch.eager on Conversation.

LG, feel free to merge, but don't close the issue.  I'll commit and close when I get to the meeting today.

**Comment by dge9788** [comment]
Works for now. Will need to be tweaked to use the "Recipient" field whenever it gets moved. Leaving issue open.

**Comment by rpk7524** [comment]
Closing, fixed in 2e933bd.

**Comment by dge9788** [comment]
Confirmed working.
